sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/ResourceManagerThrottlingInfo.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.resources.fluentcore.utils;
+
+import com.azure.core.http.HttpHeaders;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * The class to collect all throttling info from response header
```

Comment:
If possible, get info on whether network/storage has limit but does not return in headers.

Reply:
I didn't see the network/storage have specific header for throttling.

Reply:
I know. I mean if other service has limitation but not visible in header, we should mention it in javadoc (that this util tries to help but in some case not able to).